deploy/charts/trust-manager/values.yaml

```diff
@@ -239,6 +239,11 @@ app:
     # before installing trust-manager.
     namespace: cert-manager
 
+  # List of target namespaces that trust-manager can write to. By default, trust-manager can write targets in any namespace.
+  # +docs:property
+  # targetNamespaces: ["ns-1", "ns-2"]
+  targetNamespaces: []
```

Comment:
```suggestion
```

There should be no default value. The default/common behavior is documented above. An empty array default can be very confusing.